hack/README.md

```diff
@@ -60,3 +41,19 @@ For example, the following command will add a metadata section for a nested obje
 +        metadata:
 +          description: 'Standard object''s metadata. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata'
 ```
+* Update the restoreSize property to string in snapshot.storage.k8s.io_volumesnapshots.yaml  
+
+```bash
+    
+@@ -160,7 +157,7 @@ spec:
+                 of a snapshot is unknown.
+               type: boolean
+             restoreSize:
+-              - type: string
++              type: string
```

Comment:
You should show the diff between the generated results and the modified results.  The generated results should have anyOf:  - type: integer
              - type: string

```
                of a snapshot is unknown.
              type: boolean
            restoreSize:
              anyOf:
              - type: integer
              - type: string
              type: string
              description: restoreSize represents the complete size of the snapshot
```